metaci/release/migrations/0003_populate_releases.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-08-15 22:48
+from __future__ import unicode_literals
+
+import re
+
+from datetime import datetime
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations
+from github3 import login
+
+
+def populate_releases(apps, schema_editor):
+    Build = apps.get_model('build.Build')
+    Release = apps.get_model('release.Release')
+
+    release_tags = {}
+    for build in Build.objects.filter(plan__role = 'release_test'):
+        if build.branch.repo.id not in release_tags:
+            release_tags[build.branch.repo.id] = {
+                'repo': build.branch.repo,
+                'tags': set(),
+            }
+        release_tags[build.branch.repo.id]['tags'].add(build.branch.name.replace('tag: ',''))
+  
+    github = login(settings.GITHUB_USERNAME, settings.GITHUB_PASSWORD)
+    for repo, info in release_tags.items():
+        repo = github.repository(info['repo'].owner, info['repo'].name) 
```

Comment:
you're redefining repo here from an integer of the id, to an instance of a gh repo object. seems like maybe we should just iterate release_tags.values()